models/DataObject/ClassDefinition/Data/RgbaColor.php

```diff
@@ -273,7 +273,11 @@ public function getForCsvExport(DataObject\Localizedfield|DataObject\Fieldcollec
     {
         $data = $this->getDataFromObjectParam($object, $params);
 
-        return $this->getDataForEditmode($data, $object, $params) ?? '';
+        if ($data instanceof Model\DataObject\Data\RgbaColor) {
+            return sprintf('#%02x%02x%02x%02x', $data->getR(), $data->getG(), $data->getB(), $data->getA());
+        }
```

Comment:
`getDataForEditmode` does the same transformation. right?

Reply:
Yes, should we make a private method for both?

Reply:
I'm thinking of skip passing these params to `getDataForEditmode` since they are default and not used for the transformation. wdyt?

Reply:
Or should we use the getDataForEditmode method and remove the $object and $params params?

Reply:
yes, just had the same thought 😊

Reply:
@dvesh3 Done https://github.com/pimcore/pimcore/pull/15763/commits/72d1392f48b57abe1e1f1f1eed24bb75e486fc8f